packages/tooltip/src/components/tooltip/Tooltip.module.css

```diff
@@ -0,0 +1,157 @@
+.noPadding :global(.tippy-content) {
+  padding: 0;
+}
+
+:global(.tippy-box[data-theme~="light"]) {
+  box-shadow: var(--swui-shadow-popover);
+}
+
+:global(.tippy-box[data-theme~="outlined"]) {
+  border: 1px solid var(--lhds-color-blue-500);
+}
+
+:global(.tippy-box[data-theme~="info"]) {
```

Comment:
My copy-paste sense is tingling. Explain to me what any of these are good for. It doesn't seem to me that they should be used in this component at all.